site/docs/src/json/user-comments/search-response.json

```diff
@@ -0,0 +1,12 @@
+{
+  "userComments": [
+    {
+      "comment": "Not sure if this user is violating any rules or not.",
+      "commenterId": "00000000-0000-0000-0000-000000000002",
```

Comment:
Again, late to the party, but I can see wanting to search on the `commenterId`. Should I file an issue for that?

Reply:
I can add this.